packages/terra-clinical-data-grid/src/DataGrid.jsx

```diff
@@ -0,0 +1,1200 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import memoize from 'memoize-one';
+import { polyfill } from 'react-lifecycles-compat';
```

Comment:
Interesting!...Should we log an issue to update all components utilizing new React Features to ensure compatibility?

Reply:
I included this so I could use getDerivedStateFromProps (rather than componentWillReceiveProps) on React 16.2. I'm not sure we need to update anything else until React actually kills those API's with a React 17 release.

Reply:
we should be able to remove this polyfil now